Resources/Prototypes/Entities/Clothing/OuterClothing/softsuits.yml

```diff
@@ -0,0 +1,61 @@
+#base EVA suit movespeed is 0.8
```

Comment:
dev comment? also separate these by role like you did suit storage

Reply:
i figured it would be useful to have there since it's a good frame of reference for anyone who wants to add a softsuit. i'll separate them though